source/_integrations/xiaomi_miio.markdown

```diff
@@ -1411,6 +1411,23 @@ automation:
         segments: 1
 ```
 
+Single segment with repetition:
```

Comment:
This might need a bit more explanation? I think the PR description explained it really well, while in the resulting documentation, I'm not sure what I would have been looking at.

```suggestion
Single-segment with repetition:
```

Reply:
Thank you for your review! I will add more info into documentation